website/docs/general/sidebars.json

```diff
@@ -1,14 +1,14 @@
 {
   "docs": {
-    "General": [
-      "security",
-      "2fa",
-      "subscribe-guide",
+    "Community": [
```

Comment:
Hi, I would prefer keeping this as General because it contains other topics contents.

![image](https://user-images.githubusercontent.com/2106987/155283386-198e4296-35f5-4f17-9460-6b309cd8676d.png)